Gemfile.lock

```diff
@@ -82,12 +83,13 @@ GEM
     notiffany (0.1.3)
       nenv (~> 0.1)
       shellany (~> 0.0)
-    oauth2 (1.4.9)
+    oauth2 (2.0.0)
       faraday (>= 0.17.3, < 3.0)
       jwt (>= 1.0, < 3.0)
-      multi_json (~> 1.3)
       multi_xml (~> 0.5)
       rack (>= 1.2, < 3)
+      rash_alt (>= 0.4, < 1)
```

Comment:
oauth2 relaxation looks good but what are these other gems used for? `rash_alt `, `version_gem`, `hashie`

Reply:
These are all dependencies or sub-dependencies introduced in `oauth2` v2.0.0. For example: https://github.com/oauth-xx/oauth2/pull/575

Reply:
Oh I see now. LGTM. Will approve and merge

Reply:
@jv-asana Thanks! Would appreciate a new tag since this is the one gem that is holding us back from being able to upgrade `oauth2`.

Reply:
@stanhu done. Hope this unblocks you :)